### ISSUE TITLE
ductus arteriosus aneurysm

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6986]
**Preferred term label:**
Ductus arteriosus aneurysm

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal outpouching or sac-like dilatation of the ductus arteriosus. Ductus arteriosus aneurysm (DAA) is likely to emerge in the third trimester from the aortic junction of the DA, extending towards its pulmonary end.
PMID: 29656824
PMID: 27433265

Comment: Ductus arteriosus aneurysm (DAA) can only be observed postnatally if patent ductus arteriosus (PDA) is present.

**Parent term (use hpo.jax.org/app)**
Vascular dilatation HP:0002617
-- add abnormality of ductus arteriosus near this term Patent ductus arteriosus (some reorganizing appeared needed).

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:610168  3/9  (LDS2)
 OMIM:609192  0/3 (LDS1)
both  PMID: 15731757

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@JeroenBreckpot, @DominicIliescu.  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by JeroenBreckpot** [comment]
PMID: 10898444

Ductus Arteriosus Aneurysm (DAA) is a saccular dilatation of the ductus arteriosus. DAA can be either congenital or acquired (e.g. as a complication of surgical closure of a patent ductus arteriosus). Although the majority of patients with congenital DAA are asymptomatic and have a benign course, severe complications, such as rupture or thromboembolism, can occur. DAA has been observed in patients with connective tissue disorders like Marfan and Ehlers-Danlos syndrome and in patients with chromosomal syndromes (e.g. Williams, Larsen, aneuploidies).

Synonyms: aneurysm of the ductus arteriosus, ductus arteriosus aneurysm

BW
Jeroen

**Comment by DominicIliescu** [comment]
I agree with the previous definition and description.

**Comment by pnrobinson** [comment]
updated definition and synonym, thanks!
